babel.config.js

```diff
@@ -11,5 +11,5 @@ module.exports = {
     '@babel/preset-typescript',
     'module:metro-react-native-babel-preset',
   ],
-  plugins: ['@babel/plugin-proposal-class-properties'],
+  plugins: ['@babel/plugin-proposal-class-properties', './plugin.js'],
```

Comment:
It is because Jest hasn't run Reanimated plugin before tests. @Szymon20000 What you think, can I do this in this way?

Reply:
Sure